### PR TITLE
28 dashboard on click on weather widget change the city

### PR DESCRIPTION
**Pull request by Jibea**

### CODE REVIEW

File: SurvivorNext/Components/WeatherWidget.js [hunk 1]
```diff
@@ -16,7 +15,7 @@ const WeatherWidget = ({ city="Paris", apiKey="95c3a0ad67e9a863db54f6d882f56804"
       .catch((error) => {
         console.error('Error fetching weather data: ', error);
       });
-  }, [city, apiKey]);
+
```
Comment (MATIFIREofficiel) [inline SurvivorNext/Components/WeatherWidget.js:18]:
```suggestion
```

File: SurvivorNext/Pages/DashboardPage.js [hunk 2]
```diff
@@ -30,47 +65,63 @@ export default function DashboardPage() {
...
+
+
+  // const renderItemDashboard = useCallback(({ item }) => {
+  //   const Widget = item;
+  //   return (
+  //     <View style={styles.itemContainer}>
+  //       {Widget.name === "WeatherWidget" ? <UserWeatherWidget item={item} /> : <Widget />}
+  //     </View>
+  //   );
+  // }, []);
```
Comment (MATIFIREofficiel) [inline SurvivorNext/Pages/DashboardPage.js:89]:
```suggestion
```

File: SurvivorNext/Pages/DashboardPage.js [hunk 3]
```diff
@@ -79,10 +130,11 @@ export default function DashboardPage() {
         ref={bottomSheetRef}
         index={-1}
         snapPoints={snapPoints}
-        onChange={handleSheetChanges}
+        // onChange={handleSheetChanges}
```
Comment (MATIFIREofficiel) [inline SurvivorNext/Pages/DashboardPage.js:133]:
```suggestion
```

File: SurvivorNext/Pages/DashboardPage.js [hunk 4]
```diff
@@ -79,10 +130,11 @@ export default function DashboardPage() {
...
         detached={true}
         enablePanDownToClose={true}
-        handleIndicatorPress={handleIndicatorPress}
+        // handleIndicatorPress={handleIndicatorPress}
```
Comment (MATIFIREofficiel) [inline SurvivorNext/Pages/DashboardPage.js:136]:
```suggestion
```

File: SurvivorNext/Pages/DashboardPage.js [hunk 5]
```diff
@@ -120,4 +173,15 @@ const styles = StyleSheet.create({
...
     right: 20,
   },
+
+  scrollContainer: {
+    flexDirection: 'row', // Utilisez une disposition en ligne pour les widgets
+    flexWrap: 'wrap', // Permet aux widgets de passer à la ligne si nécessaire
+    justifyContent: 'space-between',
+  },
+  itemContainerScroll: {
+    width: '48%', // Largeur de chaque widget
```
Comment (MATIFIREofficiel) [inline SurvivorNext/Pages/DashboardPage.js:183]:
```suggestion
    flexDirection: 'row',
    flexWrap: 'wrap',
    justifyContent: 'space-between',
  },
  itemContainerScroll: {
    width: '48%',
```

**Review by MATIFIREofficiel — APPROVED** [review]
Good but just accept my suggestions

**Review by TurpaultM — APPROVED** [review]
ok

**Review by MATIFIREofficiel — APPROVED** [review]
Good

**Pull request merged**